README.md

```diff
@@ -4471,6 +4473,41 @@ $ curl https://mainnet.aeternity.io/mdw/v2/accounts/ak_2nVdbZTBVTbAet8j2hQhLmfNm
       },
       "type": "SpendTxEvent",
       "height": 502033
+    },
+    {
+      "height": 659373,
+      "payload": {
+        "block_hash": "mh_MXVb7wmE1tqeA2xSPhTTksLy7DE5PvR8nsu5haC2fTGpgxxhR",
+        "call_tx_hash": "th_Ugtejdn7SkJHXkC3VSdCm2SnXGgPxHgUphBneMqgR3gniZzDN",
+        "call_txi": 34224137,
+        "contract_id": "ct_2AfnEfCSZCTEkxL5Yoi4Yfq6fF7YapHRaFKDJK3THMXMBspp5z",
+        "contract_tx_hash": "th_6memqAr5S3UQp1pc4FWXT8xUotfdrdUFgBd8VPmjM2ZRuojTF",
+        "contract_txi": 8392766,
+        "function": "Oracle.query",
```

Comment:
each `IntContractCallEvent` has the `function` attribute, right?

Reply:
That's correct